OrbitGl/ProcessesDataView.cpp

```diff
@@ -102,75 +96,37 @@ void ProcessesDataView::DoSort() {
 }
 
 //-----------------------------------------------------------------------------
-void ProcessesDataView::OnSelect(int a_Index) {
-  m_SelectedProcess = GetProcess(a_Index);
-  if (!m_IsRemote) {
-    m_SelectedProcess->ListModules();
-  } else {
+void ProcessesDataView::OnSelect(int index) {
+  auto selected_process = GetProcess(index);
```

Comment:
I'm not a fan of `auto` here and below. It's not clear that `selected_process` is a `std::shared_ptr<Process>`.

Reply:
done.